controller/Lists.php

```diff
@@ -71,32 +70,27 @@ class Lists extends tao_actions_CommonModule
     private $isListsDependencyEnabled;
 
     /**
-     * Show the list of users
+     * Show the list
      * @return void
      */
     public function index()
     {
         tao_helpers_Scriptloader::addCssFile(Template::css('lists.css', 'tao'));
         $this->defaultData();
 
-        $myAdderFormContainer = new tao_actions_form_List();
-        $myForm = $myAdderFormContainer->getForm();
-
-        if ($myForm->isSubmited()) {
-            if ($myForm->isValid()) {
-                $values = $myForm->getValues();
-                $newList = $this->getListService()->createList($values['label']);
-                $i = 0;
-                while ($i < $values['size']) {
-                    $this->getListService()->createListElement($newList, __('element') . ' ' . ($i + 1));
-                    $i++;
-                }
+        $newListFormContainer = new CreateListForm();
+        $newListForm = $newListFormContainer->getForm();
+
+        if ($newListForm->isSubmited()) {
+            if ($newListForm->isValid()) {
```

Comment:
I single if does the work now, no?

```suggestion
        if ($newListForm->isSubmited() && $newListForm->isValid()) {
```

Reply:
Or https://github.com/oat-sa/extension-tao-backoffice/pull/114/files#r756419714

Reply:
Conditionals merged in 513ed7c0a2b43885737b05975d78fb23097e1df6